imapclient/util.py

```diff
@@ -17,3 +18,9 @@ def to_bytes(s, charset='ascii'):
     if isinstance(s, text_type):
         return s.encode(charset)
     return s
+
+
+class SocketTimeout(namedtuple("SocketTimeout", "connect read")):
```

Comment:
I am not sure this class really belongs in `util`, I would rather see it in the main `imapclient.imapclient`.